build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
 }
 
 plugins {
-    id "org.jetbrains.intellij" version "0.4.8"
+    id "org.jetbrains.intellij" version "0.7.3"
```

Comment:
This and the changed Gradle versions are the last version before migration steps will be required to upgrade.